data/soft-nep.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "SoftNEP",
+  "establishedAt": 1508298452,
+  "location": "Bhimsen Gola Marg, Min Bhawan, Kathmandu",
+  "website": "https://www.softnep.com",
+  "email": "mkt@softnep.com",
+  "phone": "+977 98188 61321",
```

Comment:
The format of the phone number doesn't seem correct to me.
Please refer to other files for reference!